src/defaults.json

```diff
@@ -0,0 +1,15 @@
+{
+  "spacingNames": {
+    "XS": "4px",
+    "S/2": "4px",
+    "S": "8px",
+    "M/2": "8px",
+    "M": "16px",
+    "L/2": "12px",
+    "L": "24px",
+    "XL/2": "16px",
+    "XL":"32px",
+    "XXL/2": "24px",
+    "XXL":"48px"
```

Comment:
These values should be multiples of base (see comment below), that way we can just say `"XS": 0.5` and so on

Reply:
I thought the idea was to eventually get rid of base entirely and switch over to using only aliased spacing names.  Is that not the case?

Reply:
So one disadvantage of doing it this way is that we'd have to write base={4} for every component that uses aliases until we completely switch over to this api. 

```
<Col base={4} margin={['XS', 'S/2']} padding="S,M/2">
  My paddings and margins are set using spacingNames!
</Col>
```

Reply:
I could also pass base through the provider.

Reply:
You don't have to pass base to col, `24` the same way you don't pass the multiplier. You can rely on the default value. I agree that we can eventually set it through provider but you can also write a wrapper component that changes the defaults it if needed. `XS` should really just be an alias for the number you would type otherwise, in this case `0.5`